[32m2022-10-10 10:53:57.511[0m | [31m[1mERROR   [0m | [36m__main__[0m:[36merror_page[0m:[36m60[0m - [31m[1m404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.[0m
[33m[1mTraceback (most recent call last):[0m

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\[0m[32m[1mthreading.py[0m", line [33m937[0m, in [35m_bootstrap[0m
    [1mself[0m[35m[1m.[0m[1m_bootstrap_inner[0m[1m([0m[1m)[0m
    [36mâ”‚    â”” [0m[36m[1m<function Thread._bootstrap_inner at 0x00000240264EE8B0>[0m
    [36mâ”” [0m[36m[1m<Thread(Thread-5, started daemon 21424)>[0m

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\[0m[32m[1mthreading.py[0m", line [33m980[0m, in [35m_bootstrap_inner[0m
    [1mself[0m[35m[1m.[0m[1mrun[0m[1m([0m[1m)[0m
    [36mâ”‚    â”” [0m[36m[1m<function Thread.run at 0x00000240264EE5E0>[0m
    [36mâ”” [0m[36m[1m<Thread(Thread-5, started daemon 21424)>[0m

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\[0m[32m[1mthreading.py[0m", line [33m917[0m, in [35mrun[0m
    [1mself[0m[35m[1m.[0m[1m_target[0m[1m([0m[35m[1m*[0m[1mself[0m[35m[1m.[0m[1m_args[0m[1m,[0m [35m[1m**[0m[1mself[0m[35m[1m.[0m[1m_kwargs[0m[1m)[0m
    [36mâ”‚    â”‚        â”‚    â”‚        â”‚    â”” [0m[36m[1m{}[0m
    [36mâ”‚    â”‚        â”‚    â”‚        â”” [0m[36m[1m<Thread(Thread-5, started daemon 21424)>[0m
    [36mâ”‚    â”‚        â”‚    â”” [0m[36m[1m(<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr...[0m
    [36mâ”‚    â”‚        â”” [0m[36m[1m<Thread(Thread-5, started daemon 21424)>[0m
    [36mâ”‚    â”” [0m[36m[1m<bound method ThreadingMixIn.process_request_thread of <werkzeug.serving.ThreadedWSGIServer object at 0x000002403F4A64F0>>[0m
    [36mâ”” [0m[36m[1m<Thread(Thread-5, started daemon 21424)>[0m

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\[0m[32m[1msocketserver.py[0m", line [33m683[0m, in [35mprocess_request_thread[0m
    [1mself[0m[35m[1m.[0m[1mfinish_request[0m[1m([0m[1mrequest[0m[1m,[0m [1mclient_address[0m[1m)[0m
    [36mâ”‚    â”‚              â”‚        â”” [0m[36m[1m('127.0.0.1', 57787)[0m
    [36mâ”‚    â”‚              â”” [0m[36m[1m<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...[0m
    [36mâ”‚    â”” [0m[36m[1m<function BaseServer.finish_request at 0x0000024026CE7550>[0m
    [36mâ”” [0m[36m[1m<werkzeug.serving.ThreadedWSGIServer object at 0x000002403F4A64F0>[0m

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\[0m[32m[1msocketserver.py[0m", line [33m360[0m, in [35mfinish_request[0m
    [1mself[0m[35m[1m.[0m[1mRequestHandlerClass[0m[1m([0m[1mrequest[0m[1m,[0m [1mclient_address[0m[1m,[0m [1mself[0m[1m)[0m
    [36mâ”‚    â”‚                   â”‚        â”‚               â”” [0m[36m[1m<werkzeug.serving.ThreadedWSGIServer object at 0x000002403F4A64F0>[0m
    [36mâ”‚    â”‚                   â”‚        â”” [0m[36m[1m('127.0.0.1', 57787)[0m
    [36mâ”‚    â”‚                   â”” [0m[36m[1m<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...[0m
    [36mâ”‚    â”” [0m[36m[1m<class 'werkzeug.serving.WSGIRequestHandler'>[0m
    [36mâ”” [0m[36m[1m<werkzeug.serving.ThreadedWSGIServer object at 0x000002403F4A64F0>[0m

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\[0m[32m[1msocketserver.py[0m", line [33m747[0m, in [35m__init__[0m
    [1mself[0m[35m[1m.[0m[1mhandle[0m[1m([0m[1m)[0m
    [36mâ”‚    â”” [0m[36m[1m<function WSGIRequestHandler.handle at 0x0000024026EEF9D0>[0m
    [36mâ”” [0m[36m[1m<werkzeug.serving.WSGIRequestHandler object at 0x000002403F507E20>[0m

  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\werkzeug\serving.py", line 345, in handle
    BaseHTTPRequestHandler.handle(self)
    â”‚                      â”‚      â”” <werkzeug.serving.WSGIRequestHandler object at 0x000002403F507E20>
    â”‚                      â”” <function BaseHTTPRequestHandler.handle at 0x0000024026E1F4C0>
    â”” <class 'http.server.BaseHTTPRequestHandler'>

  File "[32mC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\http\[0m[32m[1mserver.py[0m", line [33m425[0m, in [35mhandle[0m
    [1mself[0m[35m[1m.[0m[1mhandle_one_request[0m[1m([0m[1m)[0m
    [36mâ”‚    â”” [0m[36m[1m<function WSGIRequestHandler.handle_one_request at 0x0000024026EEFB80>[0m
    [36mâ”” [0m[36m[1m<werkzeug.serving.WSGIRequestHandler object at 0x000002403F507E20>[0m

  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\werkzeug\serving.py", line 379, in handle_one_request
    return self.run_wsgi()
           â”‚    â”” <function WSGIRequestHandler.run_wsgi at 0x0000024026EEF940>
           â”” <werkzeug.serving.WSGIRequestHandler object at 0x000002403F507E20>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\werkzeug\serving.py", line 323, in run_wsgi
    execute(self.server.app)
    â”‚       â”‚    â”‚      â”” <werkzeug.debug.DebuggedApplication object at 0x000002403F455340>
    â”‚       â”‚    â”” <werkzeug.serving.ThreadedWSGIServer object at 0x000002403F4A64F0>
    â”‚       â”” <werkzeug.serving.WSGIRequestHandler object at 0x000002403F507E20>
    â”” <function WSGIRequestHandler.run_wsgi.<locals>.execute at 0x000002403F4E0A60>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\werkzeug\serving.py", line 314, in execute
    for data in application_iter:
                â”” <generator object DebuggedApplication.debug_application at 0x000002403F4A8820>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\werkzeug\debug\__init__.py", line 304, in debug_application
    app_iter = self.app(environ, start_response)
               â”‚    â”‚   â”‚        â”” <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x000002403F4BECA0>
               â”‚    â”‚   â”” {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1688>, 'wsgi.errors': <_io.TextIOW...
               â”‚    â”” <Flask 'main'>
               â”” <werkzeug.debug.DebuggedApplication object at 0x000002403F455340>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\flask\app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
           â”‚    â”‚        â”‚        â”” <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x000002403F4BECA0>
           â”‚    â”‚        â”” {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1688>, 'wsgi.errors': <_io.TextIOW...
           â”‚    â”” <function Flask.wsgi_app at 0x00000240281878B0>
           â”” <Flask 'main'>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
               â”‚    â”” <function Flask.full_dispatch_request at 0x0000024028184F70>
               â”” <Flask 'main'>
> File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
         â”‚    â”” <function Flask.dispatch_request at 0x0000024028184EE0>
         â”” <Flask 'main'>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\flask\app.py", line 1926, in dispatch_request
    self.raise_routing_exception(req)
    â”‚    â”‚                       â”” <Request 'http://127.0.0.1:5000/favicon.ico' [GET]>
    â”‚    â”” <function Flask.raise_routing_exception at 0x0000024028184E50>
    â”” <Flask 'main'>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\flask\app.py", line 1908, in raise_routing_exception
    raise request.routing_exception
          â”‚       â”” <NotFound '404: Not Found'>
          â”” <Request 'http://127.0.0.1:5000/favicon.ico' [GET]>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\flask\ctx.py", line 350, in match_request
    result = self.url_adapter.match(return_rule=True)
             â”‚    â”‚           â”” <function MapAdapter.match at 0x00000240280EA700>
             â”‚    â”” <werkzeug.routing.MapAdapter object at 0x000002403F50F100>
             â”” <RequestContext 'http://127.0.0.1:5000/favicon.ico' [GET] of main>
  File "C:\Users\xmovi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\werkzeug\routing.py", line 1945, in match
    raise NotFound()
          â”” <class 'werkzeug.exceptions.NotFound'>

[31m[1mwerkzeug.exceptions.NotFound[0m:[1m 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.[0m
